docs/how-to-guides/deployment-options.md

```diff
@@ -133,6 +133,30 @@ This [PowerShell deployment script](../deploy/Deploy-FhirConverterService.ps1) s
 
 Refer to [Configurable Settings](#configurable-settings) for more information on the required parameters to be provided and the default values used for optional parameters.
 
+### Deploy a Container App to an existing Container Apps Environment
+
+To deploy a Container App to an existing Container Apps environment, use the [Deploy-FhirConvertService.bicep](../deploy/Deploy-FhirConverterService.bicep) template using the following command:
+
+```
+az deployment group create --resource-group <Resource Group containing existing Container Apps Environment> --template-file Deploy-FhirConverterService.bicep --parameters location=<Location of Container App Environment> appName=<Container App name> envName=<Name of existing Container Apps Environment> imageTag=<Image tag from MCR> 
+```
+
+In addition to the parameters listed for the [FhirConverter-SingleAzureDeploy](../deploy/FhirConverter-SingleAzureDeploy.bicep) in [Parameters](#parameters), the Deploy-FhirConvertService bicep template takes the following parameters:
+
```

Comment:
I'm a little confused by this section. Why are these parameters separated from the above?

Reply:
I separated them because they are specific to this bicep template, rather than the single deploy template. I'll find a way to combine them so that it's clearer

Reply:
Oh got it. Can we provide a way to use the same single deploy template to setup a new service? I think we pretty much expose all the expected configs there?

Reply:
Sure, we will just need to expose these 4 additional fields. I'll add that in a separate PR so that we can get the troubleshooting changes out sooner.

Reply:
Ah got it, I thought we already would have those, In that case, its fine, can leave it the way you have it unless we get any feedback. Did you already make that change? Sorry for the confusion!

Reply:
Work item created: https://microsofthealth.visualstudio.com/Health/_workitems/edit/120425